CMakeLists.txt

```diff
@@ -248,6 +248,18 @@ if (UNIX)
     find_library(CRYPTO_LIB libcrypto.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/openssl/lib/ REQUIRED NO_DEFAULT_PATH)
     find_library(AWS_CORE_LIB libaws-cpp-sdk-core.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
     find_library(AWS_S3_LIB libaws-cpp-sdk-s3.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_AUTH_LIB libaws-c-auth.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_CAL_LIB libaws-c-cal.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_COMMON_LIB libaws-c-common.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_COMPRESSION_LIB libaws-c-compression.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_EVENT_STREAM_LIB libaws-c-event-stream.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_CHECKSUMS_LIB libaws-checksums.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_HTTP_LIB libaws-c-http.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_IO_LIB libaws-c-io.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_C_MQTT_LIB libaws-c-mqtt.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
+    find_library(AWS_CRT_CPP_LIB libaws-crt-cpp.a PATHS deps-build/${PLATFORM}/${CMAKE_BUILD_TYPE}/aws/lib/ REQUIRED NO_DEFAULT_PATH)
```

Comment:
please check [Build-Test-CodeCov-Linux (Release)](https://github.com/snowflakedb/libsnowflakeclient/actions/runs/8291055387/job/22690972743?pr=682#logs)
there is a problem
```
../../deps-build/linux/Release/aws/lib64/libaws-crt-cpp.a(Api.cpp.o): In function `Aws::Crt::ApiHandle::ApiHandle(aws_allocator*)':
Api.cpp:(.text+0x58): undefined reference to `aws_event_stream_library_init'
../../deps-build/linux/Release/aws/lib64/libaws-crt-cpp.a(Api.cpp.o): In function `Aws::Crt::ApiHandle::~ApiHandle()':
Api.cpp:(.text+0x83a): undefined reference to `aws_event_stream_library_clean_up'
```

Reply:
fixed.